BMS Workspace-git/bms-1227-fw/Phantom_drivers/drivers/include/bms_data.h

```diff
@@ -27,6 +27,8 @@ typedef struct bmsFlags
 								   // TODO: If 1, put BMS in FAULT state
 	uint8_t BAD_SLAVE_CONNECTION_FLAG; // If 1, then BMS master is unable to read/write to/from at least one BMS slave
 									   // TODO: If 1, put BMS in FAULT state
+	uint8_t OVER_CURRENT_FLAG; //if 1, then the current transducer has read over 400A instantaneous current

```

Comment:
This can be split into over and under current flags to make the fault even clearer.

Reply:
- split this function into two, one for over current case and one for under current
- named "underHVCurrentFaultCheck()" and "overHVCurrentFaultCheck"